.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Found a bug? Tell us about it!
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Steps to reproduce**
+Describe the steps to reproduce the bug, if applicable:
+
+**Environment (please complete the following information):**
+ - OS: [e.g. Ubuntu 18.04]
+ - Python version: [e.g. 3.7]
+ - Oríon version [e.g. 0.1.7]
```

Comment:
Maybe add info about the database backend used if applicable.

Reply:
Great idea!